src/makeRobot.js

```diff
@@ -37,8 +37,76 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0, // 14
+      y: 0, // 21
+    },
+
+    get info() {
+      return 'name: ' + this.name + ', chip version: '
+      + this.version + ', wheels: ' + this.wheels;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goBack(n = 1) {
+      if (n < 1) {
+        return this;
+      }
+
+      this.coords.y -= n;
+
+      return this;
+    },
+
+    goForward(n = 1) {
+      if (n < 1) {
+        return this;
+      }
+
+      this.coords.y += n;
+
+      return this;
+    },
```

Comment:
методи можна скоротити 
```suggestion
    goForward(n = 1) {
      if (n > 0) {
        this.coords.y += n;
      }

      return this;
    },
```